docs/sql-data-sources-parquet.md

```diff
@@ -278,6 +278,19 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
   </td>
   <td>1.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.sql.parquet.outputTimestampType</code></td>
```

Comment:
I think all sql confs are now documented at https://spark.apache.org/docs/latest/configuration.html#spark-sql

Reply:
@HyukjinKwon thanks! I didn't see that.

That said, anyone following the parquet guide doesn't see all the parquet options. Perhaps we change this to link back to the general config docs and state that all parquet options can be found there?

Reply:
Probably we could add a link to the SQL configuration page to avoid duplication.